src/components/macroview/rightPanel/editForms/KeyPressForm.tsx

```diff
@@ -51,12 +63,26 @@ export default function KeyPressForm({
   )
 
   const onInputBlur = useCallback(() => {
-    let duration
-    if (keypressDuration === '') {
-      duration = 0;
-    } else {
+    let duration = 20
 
-      duration = parseInt(keypressDuration)
+    if (Number(keypressDuration) >= 20) {
+      duration = Number(keypressDuration)
+    } else if (keypressDuration === '') {
```

Comment:
It's kinda nasty to keep parsing this from a string into a number

Reply:
Also, using a magic number like this here is also kinda nasty. It should be a define or a constant and used to create the strings that also reference this number

Reply:
Tweaked: [5d256ec](https://github.com/WootingKb/wooting-macros/pull/194/commits/5d256ecf2bf5fb0a8a4c1027963d6ec4b0fe7aa8)